src/server/apis/settings.ts

```diff
@@ -35,8 +34,34 @@ function getRouter(github: GitHub): express.Router {
       // pagination
       // Switching to GitHub.cursorQuery() would be best option.
 
+      const apiOrgs = [];
+      const orgsData = orgDetails.data.viewer.organizations.nodes;
+      if (orgsData) {
+        for (const org of orgsData) {
+          if (!org) {
+            continue;
+          }
+  
+          let hookEnabled = false;
+          if (org.viewerCanAdminister) {
+            const hooks = await github.get(`orgs/${org.login}/hooks`, loginDetails.token);
```

Comment:
This executes in serial. Can we change it to parallel by using Promise.all instead of await?